src/state.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Enumeration of node state types.
+ */
+export const enum State {
```

Comment:
Possibly a breaking change: I've changed `State` to a regular old TypeScript enum rather than the symbols that were being used before. I'm not sure if/how external code would be impacted.

Reply:
I don't think it's going to be much of a problem. Most people would just import the different states to use them as the result of action function calls.